### PR TITLE
Adds Google Cloud Run and GraalVM Category

### PR DESCRIPTION
**Pull request by sdelamo**
- Google Cloud Run Category 
-  GraalVM Category
- Instructions to cleanup gcp projects
- Update Images

### CODE REVIEW

File: src/docs/images/aws-lambda-function-graalvm-warm-startup.png
Comment (timyates) [inline src/docs/images/aws-lambda-function-graalvm-warm-startup.png:1]: This image doesn't seem to be used anywhere...

File: guides/mn-application-aws-lambda-graalvm/mn-application-aws-lambda-graalvm.adoc [hunk 1]
```diff
@@ -16,13 +16,17 @@ common:completesolution.adoc[]
...
+
 The generated application contains a `HomeController`. It responds to POST request to `/`.
 
 source:HomeController[]
 
 callout:controller[arg0=/]
 * The `@Get` annotation maps HTTP requests to `/` to the `index` method.
 
+== Controller Test
+
```
Comment (timyates) [inline guides/mn-application-aws-lambda-graalvm/mn-application-aws-lambda-graalvm.adoc:29]: This guide has a screenshot that shows it taking 1s for a Graal native lambda to be executed...

I assume this is wrong?

**Review by timyates — CHANGES_REQUESTED** [review]

File: guides/mn-application-aws-lambda-graalvm/mn-application-aws-lambda-graalvm.adoc (hunk 1 above)
Comment (sdelamo) [reply guides/mn-application-aws-lambda-graalvm/mn-application-aws-lambda-graalvm.adoc:29]: No, it was what I got with a cold startup and 128Mb of memory. I have changed it use the warm startup screenshot instead

File: guides/mn-serverless-function-aws-lambda-graalvm/mn-serverless-function-aws-lambda-graalvm.adoc [hunk 2]
```diff
@@ -86,7 +86,7 @@ However, I like to specify it in the console as well:
 
 `example.micronaut.FunctionRequestHandler`
 
-image::lambda-custom-runtime-bookrequest-handler.png[]
+image::lambda-custom-runtime-functionrequest-handler.png[]
```
Comment (timyates) [inline guides/mn-serverless-function-aws-lambda-graalvm/mn-serverless-function-aws-lambda-graalvm.adoc:89]: I think lambda-custom-runtime-bookrequest-handler.png can now be deleted?

**Review by timyates — APPROVED** [review]

File: guides/mn-serverless-function-aws-lambda-graalvm/mn-serverless-function-aws-lambda-graalvm.adoc (hunk 2 above)
Comment (sdelamo) [reply guides/mn-serverless-function-aws-lambda-graalvm/mn-serverless-function-aws-lambda-graalvm.adoc:89]: deleted here https://github.com/micronaut-projects/micronaut-guides/pull/1342/commits/06b72abea05394b2437fa085af613fb10333b2d8

**Pull request merged**